src/components/MoreInfo.vue

```diff
@@ -557,6 +557,7 @@ export default {
         };
         this.$store.dispatch("doReport", report);
       }
+      // this.$root.$emit("reloadTable")
```

Comment:
then it does not recharge when it is reported?

Reply:
I edited this, it does now